java.lang.NoClassDefFoundError: org/tukaani/xz/XZInputStream

When opening a CRAM file created with `use_lzma=1` in IGV, the following error occurs.

```
SEVERE [LongRunningTask] java.lang.NoClassDefFoundError: org/tukaani/xz/XZInputStream
        at htsjdk@4.3.0-module/htsjdk.samtools.cram.compression.LZMAExternalCompressor.uncompress(LZMAExternalCompressor.java:57)
        at htsjdk@4.3.0-module/htsjdk.samtools.cram.structure.block.Block.getUncompressedContent(Block.java:232)
        at htsjdk@4.3.0-module/htsjdk.samtools.cram.structure.SliceBlocksReadStreams.<init>(SliceBlocksReadStreams.java:67)
        at htsjdk@4.3.0-module/htsjdk.samtools.cram.encoding.reader.CramRecordReader.<init>(CramRecordReader.java:89)
        at htsjdk@4.3.0-module/htsjdk.samtools.cram.structure.Slice.deserializeCRAMRecords(Slice.java:420)
        at htsjdk@4.3.0-module/htsjdk.samtools.cram.structure.Container.getSAMRecords(Container.java:320)
        at htsjdk@4.3.0-module/htsjdk.samtools.CRAMIterator.nextContainer(CRAMIterator.java:112)
        at htsjdk@4.3.0-module/htsjdk.samtools.CRAMIterator.hasNext(CRAMIterator.java:204)
        at htsjdk@4.3.0-module/htsjdk.samtools.SamReader$AssertingIterator.hasNext(SamReader.java:608)
        at org.igv/org.broad.igv.sam.reader.WrappedIterator.hasNext(WrappedIterator.java:49)
        at org.igv/org.broad.igv.sam.AlignmentUtils.firstAlignments(AlignmentUtils.java:243)
        at org.igv/org.broad.igv.sam.AlignmentDataManager.inferType(AlignmentDataManager.java:385)
        at org.igv/org.broad.igv.sam.AlignmentDataManager.<init>(AlignmentDataManager.java:91)
        at org.igv/org.broad.igv.track.TrackLoader.loadAlignmentsTrack(TrackLoader.java:898)
        at org.igv/org.broad.igv.track.TrackLoader.load(TrackLoader.java:198)
        at org.igv/org.broad.igv.ui.IGV.load(IGV.java:1277)
        at org.igv/org.broad.igv.ui.IGV.lambda$load$6(IGV.java:1319)
        at org.igv/org.broad.igv.util.LongRunningTask.call(LongRunningTask.java:73)
        at org.igv/org.broad.igv.util.LongRunningTask.call(LongRunningTask.java:43)
        at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:328)
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1090)
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:614)
        at java.base/java.lang.Thread.run(Thread.java:1474)
Caused by: java.lang.ClassNotFoundException: org.tukaani.xz.XZInputStream
        at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:580)
        at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:490)
        ... 23 more
```

Environment
```
INFO [Main] Startup  IGV Version 2.19.5 07/09/2025 01:47 PM
INFO [Main] Java 25 (build 25+36-LTS) 2025-09-16
INFO [Main] Java Vendor: Microsoft https://www.microsoft.com
INFO [Main] JVM: OpenJDK 64-Bit Server VM Microsoft-12398178
INFO [Main] OS: Windows Server 2022 10.0 amd64
```

--- a/build.gradle
+++ b/build.gradle
@@ -59,7 +59,6 @@ configurations {
 
         exclude group: 'xmlpull', module: 'xmlpull'
         exclude group: 'xpp3', module: 'xpp3_min'
-        exclude group: 'org.tukaani', module: 'xz'
 
         exclude group: 'org.apache.ant', module: 'ant'
         exclude group: 'org.apache.ant', module: 'ant-launcher'

